### PR TITLE
Changes to review service of CSP

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed eclipse-xpanse/xpanse#2112

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

**Comment by yuanxiaojun987** [comment]
LGTM

**Comment by luoping-12345** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

File: src/components/content/deployedServices/common/DeployedServicesHostingType.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export function DeployedServicesHostingType({
     currentServiceHostingType,
     className,
 }: {
-    currentServiceHostingType: serviceHostingType;
+    currentServiceHostingType: 'self' | 'service-vendor';
```
Comment (swaroopar) [inline src/components/content/deployedServices/common/DeployedServicesHostingType.tsx:15]: we must not define the types again. we must read the values only from the API generated classes.

**Review by swaroopar — CHANGES_REQUESTED** [review]
looks good. just one small change required.

**Pull request closed without merging**